nilearn/datasets/func.py

```diff
@@ -24,13 +33,22 @@ def fetch_haxby_simple(data_dir=None, url=None, resume=True, verbose=1):
         Path of the data directory. Used to force data storage in a specified
         location. Default: None
 
+    resume: bool
+        Whether to resume download of a partly-downloaded file.
+
+    query_server: bool, optional (default: True)
+        if False, then only cached data is used.
```

Comment:
I'd rather not add this to all the downloaders. It's going to lead to subtle bugs in what people are doing. I'd rather have the download just crash if there is no internet.